sig-policy/contribution-guidelines.md

```diff
@@ -74,3 +74,10 @@ Once the PR is approved, Prow will merge the PR.
 
 For more details on how the code review process works with Prow, you may read the
 [Kubernetes code review process](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process).
+
+### Reviews
+
+All design document PRs and PRs that add new functionality must have at least two approvals from the Policy SIG before
```

Comment:
From whom in the Policy SIG? Can I be one of the approvals for my own new functionality?